recipes/toastedmarshmallow/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "toastedmarshmallow" %}
+{% set version = "0.2.6" %}
+{% set sha256 = "15dee79cac691c0a1dc9df694b6e014d00d465b588f49b893c9ece27d09856a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
You can start dropping this line. Latest conda can build the `fn` from the `url`.